src/Miscellaneous/PostJumper.coffee

```diff
@@ -0,0 +1,52 @@
+PostJumper = 
+  init: ->
+    @capcodeMap  = new Map()
+    @uniqueIDMap = new Map()
```

Comment:
It doesn't look like you're using any `Map` specific features, in which case it would be better to use a plain object.